plugins/csv/tests/output.rs

```diff
@@ -0,0 +1,276 @@
+
+    let expected_string_no_late_attributes = indoc! {
+        r#"metric;timestamp;value;resource_kind;resource_id;consumer_kind;consumer_id;attributes_1;__late_attributes
+           test_metric_u64;1970-01-01T00:00:00Z;0;local_machine;;local_machine;;value1;
+           test_metric_u64;1970-01-01T00:00:00Z;1;local_machine;;local_machine;;;
+           test_metric_f64;1970-01-01T00:00:00Z;0.5;local_machine;;local_machine;;;
+           test_metric_f64;1970-01-01T00:00:00Z;0.75;local_machine;;local_machine;;value1;
+        "#
+    };
+    let expected_string_with_late_attributes = indoc! {
+        r#"metric;timestamp;value;resource_kind;resource_id;consumer_kind;consumer_id;attributes_1;__late_attributes
+           test_metric_u64;1970-01-01T00:00:00Z;0;local_machine;;local_machine;;value1;
+           test_metric_u64;1970-01-01T00:00:00Z;1;local_machine;;local_machine;;;
+           test_metric_f64;1970-01-01T00:00:00Z;0.5;local_machine;;local_machine;;;
+           test_metric_f64;1970-01-01T00:00:00Z;0.75;local_machine;;local_machine;;value1;
+           test_metric_u64;1970-01-01T00:00:00Z;0;local_machine;;local_machine;;;late_attributes_1=value1, late_attributes_2=value2
+        "#
+    };
+
+    let elastic_output = OutputName::from_str("csv", "out");
+    let runtime_expectations = RuntimeExpectations::new()
+        .create_metric::<u64>("test_metric_u64", Unit::Unity)
+        .create_metric::<f64>("test_metric_f64", Unit::Unity)
+        .test_output(
+            elastic_output.clone(),
+            {
+                move |ctx: &mut OutputCheckInputContext| -> MeasurementBuffer {
+                    // Prepare input ( multiple points with/without attributes u64 or f64)
+                    let metrics = TestMetrics::get(ctx);
+                    let point1 = simple_point(metrics.metric_u64, WrappedMeasurementValue::U64(0))
+                        .with_attr("attributes_1", "value1");
+                    let point2 = simple_point(metrics.metric_u64, WrappedMeasurementValue::U64(1));
+
+                    let point3 = simple_point(metrics.metric_f64, WrappedMeasurementValue::F64(0.5));
+                    let point4 = simple_point(metrics.metric_f64, WrappedMeasurementValue::F64(0.75))
+                        .with_attr("attributes_1", "value1");
+
+                    // Give input to the pipeline
+                    MeasurementBuffer::from(vec![point1, point2, point3, point4])
+                }
+            },
+            {
+                move || {
+                    assert_eq!(
+                        fs::read_to_string(result_file.as_str()).unwrap(),
+                        expected_string_no_late_attributes
+                    );
+                }
+            },
+        )
+        // Adding another point to have late attributes
+        .test_output(
+            elastic_output.clone(),
+            {
+                move |ctx: &mut OutputCheckInputContext| -> MeasurementBuffer {
```

Comment:
Can this be written
```suggestion
                move |ctx| {
```
?